guides/v2.2/cloud/configure/setup-cron-jobs.md

```diff
@@ -54,6 +54,9 @@ MAILTO=""
 * * * * *           cronrun
 ```
 
+{: .bs-callout-tip}
```

Comment:
@atwixfirster  Good clarification here.  I think it would be better to add this as a note after the opening paragraph for the "Review cron configuration" 

So it would be {: .bs-callout-info } instead of "tip".

Reply:
**moved**